tfhe/src/integer/server_key/radix_parallel/scalar_add.rs

```diff
@@ -682,8 +682,7 @@ impl ServerKey {
                         (high_block + high_scalar_block) % self.message_modulus().0 as u64;
                     high_block_result <<= 2;
 
-                    (high_block_result + (low_block_state << 1))
-                        % (self.message_modulus().0 * 4) as u64
+                    (high_block_result + (low_block_state << 1)) % packed_modulus
```

Comment:
:joy:

Reply:
🧐 did I miss something ?

Reply:
Yes there was a *4 in there before

Reply:
worth looking for * 4 or 4 * in the code base for this sort of things